packages/manager/apps/web/client/app/app.js

```diff
@@ -40,7 +40,7 @@ import config from './config/config';
 import domainEmailObfuscation from './domain/email-obfuscation/index';
 import domainOptin from './domain/optin/index';
 import domainDnsZone from './dns-zone';
-import hostingWebsiteCoach from './hosting/website-coach/website-coach.module';
+import hosting from './hosting/hosting.module';
```

Comment:
```suggestion
import hosting from './hosting/hosting';
```
? Isn't the `index.js` the lazy loading ?

Reply:
I don't understand your suggestion, there is no lazy-loading or `index.js` file here.
Have you an example of what you want to be achieved ?

Reply:
@FredericEspiau I don't understand your suggestion either 😄 
I'm not sure that your suggestion can be achieved. If not targetting `index.js` I think you have to specify the entire file name